### DISCUSSION TITLE
Images in posts stopped working when i changed to PaperModX

### DISCUSSION THREAD

**Discussion by metasikander** [reorx/hugo-PaperModX#28] [Q&A]
I'm really struggeling here, so trying to figure out where the issue is.

Website code: https://github.com/metasikander/lhote.me/blob/main/content/posts/nip-05_Simple_Guide.md
Website post: https://lhote.me/posts/nip-05_simple_guide/
Supposed to look like: https://gist.github.com/metasikander/609a538e6a03b2f67e5c8de625baed3e#file-nip-05_simple_guide-md

There are supposed to be two images in that post, but whatever i try, i get something like `WARN 2022/09/15 16:01:55 No image found for images/nip05-guide/branle1.png from posts/nip-05_Simple_Guide.md` locally with `hugo server`
I have moved the images around, tried link them from root, and much more but i'm not able to get images to work.
The cover-image and profile image on the front page works, but not markdown-links within the post.
It worked fine when i used regular PaperMod, so i wonder if something broke when i changed, but i'm also open to the possibility that i'm doing something wrong.
I'm at my wits ends here. If anyone have any tips, i'm all ears!

**Comment by MarkBTomlinson** [comment]
I am looking at the same issue, even started a new hugo site to check and still no images. Did you find a solution?

**Reply by metasikander** [reply]
no i had to go back to the base theme.

**Reply by MarkBTomlinson** [reply]
I see, thanks. I am still looking and found the images aren't getting the correct URL unless the are in the root Assets folder, not the one inder Content, this is where they do work in the original PaperMod.
I will look some more and if I figure out how to use the /Content/Assets location I will post back, but for now /Asstets/xx works.
All very odd and trying to understand where it coulld be set in the Theme is not easy.
All the best!

**Reply by metasikander** [reply]
mmm. i see.
that's good insight. thank you
